decidim-core/lib/tasks/decidim_tasks.rake

```diff
@@ -2,5 +2,27 @@
 
 namespace :decidim do
   desc "Install migrations from Decidim to the app."
-  task upgrade: ["railties:install:migrations"]
+  task upgrade: [:choose_target_plugins, :"railties:install:migrations"]
+
+  desc "Setup environment so that only decidim migrations are installed."
+  task :choose_target_plugins do
```

Comment:
I'm not sure about this change. If we're supporting Rails 5.2 now, shouldn't we allow the ActiveStorage migrations to be installed? 😕

Reply:
The original intent of `decidim:upgrade` was to `only` upgrade decidim. Maybe we could explicitly add the `ActiveStorage` ones (or write some instructions on the upgrade guide) once we make use of them, but I think it's fine the way it is now.

Reply:
Maybe I'm missing something, what this (specific) changes have to do with ActiveStorage?

Reply:
Hei! The thing is that this `activestorage` plugin is shipped as an engine with its own migrations. And this task was previously installing all migrations from all available engines (thus including `activestorage` ones).

I agree with @josepjaume's comment, but the real motivation for this change is not correctness or saving a couple of unused tables. Without this, I was not able to add new migrations to the test app, it would spit errors related to `activestorage` tables, not sure why...